### ISSUE TITLE
Hotline not reachable from abroad

### ISSUE THREAD

**Issue by tkowark** [corona-warn-app/cwa-documentation#502]
Via email, we received feedback that users abroad cannot call the 0800 number in Germany so they have to wait until coming back to Germany before being able to call the hotline.

Suggested fix: Provide an alternate number that can be called from abroad.

---
Since 17.03.2021:

Technische Hotline: +49 30 498 75401
Verifikationshotline: +49 30 498 75402

These numbers involve some costs in contrast to the 0800 numbers..

---
Internal Tracking ID: [EXPOSUREAPP-2566](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-2566)
Internal Tracking ID: [EXPOSUREAPP-5138](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5138)

---
Internal Tracking ID: [EXPOSUREAPP-5138](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5138) iOS
Internal Tracking ID: [EXPOSUREAPP-6011](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6011) Android

**Comment by dsarkar** [comment]
Internal Tracking ID: EXPOSUREAPP-2566

**Comment by dsarkar** [comment]
Website: removed +49. Stand by for an international number, please. Best, DS

**Comment by dsarkar** [comment]
@MikeMcC399 

`Is there any movement in announcing an international phone number for the TAN Hotline?`
No news here.

Inside CWA wording will change slightly and +49 removed.

**Comment by MikeMcC399** [comment]
@dsarkar 
Thanks for the update! Are you able to share what has been going on in the background and the reasons? Policy? Technical? Organisational?

**Comment by dsarkar** [comment]
@MikeMcC399 I will try to get some info.

**Comment by MikeMcC399** [comment]
@dsarkar 
If the verification hotline 0800 7540002 is a national German telephone number only, what is the advice for a user of the CWA app who is currently outside Germany and has tested positive with a PCR test?

https://www.coronawarn.app/en/faq/#test_in_other_country still says:

"Please note that currently, the hotline 0800 7540002 (as well as hotline 0800 7540001) can only be reached from Germany. We will update you soon with any further developments on this issue."

**Comment by dsarkar** [comment]
@MikeMcC399 Yes, it was discussed internally several times. We have no feedback yet. Next week we try to get the status on this one.

**Comment by dsarkar** [comment]
new ticket [EXPOSUREAPP-5138](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5138)

**Comment by dsarkar** [comment]
Both issues #501 and #502 were handled in the same ticket, splitting it now.

**Comment by GisoSchroederSAP** [comment]
This request was already raised and transmitted to the respective hotline team.
They also asked for confirmation to setup a new number that can be reached internationally - approval is pending at BMG (Federal Health Ministry) sind Dec 6th.

I'm going to ask for an update, now.

**Comment by ndegendogo** [comment]
Thanks @GisoSchroederSAP for keeping us updated!

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP

Did you get any feedback on the future of the TAN Hotline 0800 7540002 from the Bundesministerium für Gesundheit (BMG)?

The FAQ article https://www.coronawarn.app/en/faq/#test_in_other_country still has the following text posted on Jan 8, 2021 promising a further update "soon":

**UPDATE**
Please note that currently, the TAN hotline 0800 7540002 (as well as the technical hotline 0800 7540001) can only be reached from Germany. We will update you soon with any further developments on this issue.

**Comment by dsarkar** [comment]
@MikeMcC399 I have forwarded the question internally.

**Comment by GisoSchroederSAP** [comment]
Hey, meanwhile I received confirmation the new international dial-in number will be technically established in the middle of March. Once the technical setup is performed, the number get virtualized for call handling independent from any location. 
The current plan for implementation is next week, in case of delays I will be updated accordingly.
So far, I have no knowledge about the exact phone number. As soon as we get the details, we will update this issue and the FAQ accordingly.

**Comment by dsarkar** [comment]
Nummern für die Auslandserreichbarkeit (ab gestern/heute geschaltet):
Technische Hotline:        +49 30 498 75401
Verifikationshotline:       +49 30 498 75402

**Comment by MikeMcC399** [comment]
@dsarkar 
Thanks for the new numbers! Will you update https://www.coronawarn.app/ accordingly?

The 0800 numbers are on the main page and they are used in several of the FAQ articles too.

**Comment by GisoSchroederSAP** [comment]
@MikeMcC399 , sure, we will update the FAQ accordingly. 
And, obviously, those new numbers will be incorporated in the App UI as well.

**Comment by dsarkar** [comment]
FAQ entries have been updated. We close this issue once in-app info is adjusted.

**Comment by MikeMcC399** [comment]
@dsarkar 
> FAQ entries have been updated. 

Thank you!

Are you planning to change the footer of the main page?

![coronawarn app footer](https://user-images.githubusercontent.com/66998419/111656296-aee18d80-880a-11eb-938f-45b15391cba8.jpg)

Also https://www.coronawarn.app/de/faq/#available_languages still has the typo "So erreichen Sie uns außerhalb **Detschlands**"
https://github.com/corona-warn-app/cwa-website/blob/10c9b616aa0119fee8be69d5f3b1620bbb468292/src/data/faq_de.json#L416

> We close this issue once in-app info is adjusted.

Do you expect the change in 1.15 or not until 2.0?

**Comment by dsarkar** [comment]
> Detschlands

5x in the FAQ !!! Corrected it. A lot of people told me .... ;-)

> Are you planning to change the footer of the main page?
Yep, will do so.

**Comment by dsarkar** [comment]
@MikeMcC399 This issue can't make it into 1.15 anymore. So, it will be (let's be careful with such statements!) probably 2.0, which is the release after 1.15.

**Comment by MikeMcC399** [comment]
@dsarkar 
> This issue can't make it into 1.15 anymore. So, it will be (let's be careful with such statements!) probably 2.0, which is the release after 1.15.

I wasn't trying to tie you down to the release numbering. I just wanted to know really if it would be in the next release (and I thought myself anyway that it was too late). So you just confirmed that. Thank you!

**Comment by dsarkar** [comment]
PR for footer webpage https://github.com/corona-warn-app/cwa-website/pull/1000

**Comment by MikeMcC399** [comment]
@dsarkar 
> 5x in the FAQ !!! Corrected it. A lot of people told me .... ;-)

Thanks for fixing also number 6 typo! All good now!

**Comment by dsarkar** [comment]
@MikeMcC399 thanks for thorough cross-checking.

**Comment by dsarkar** [comment]
Internal Tracking ID: [EXPOSUREAPP-5138](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5138) iOS
Internal Tracking ID: [EXPOSUREAPP-6011](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6011) Android

**Comment by dsarkar** [comment]
FYI https://github.com/corona-warn-app/cwa-app-ios/pull/2370/files

**Comment by dsarkar** [comment]
FYI https://github.com/corona-warn-app/cwa-app-android/pull/2789

**Comment by MikeMcC399** [comment]
https://github.com/corona-warn-app/cwa-app-android/pull/2789 has been merged into branch [release/2.1.x](https://github.com/corona-warn-app/cwa-app-android/tree/release/2.1.x). 

The German language UI looks good. The English language translation has not yet caught up, so it would be good to check this when the first Release Candidate for 2.1.x is published here.

**Comment by Ein-Tim** [comment]
Version 2.1 has been released. Can this be closed?

**Comment by heinezen** [comment]
@Ein-Tim I think so. The tickets are completed.

Closing the issue as this has been implemented in CWA release 2.1

----
Corona-Warn-App Open Source Team

**Comment by MikeMcC399** [comment]
I checked the 2.1 Android release in English and German and all looks fine for the technical and TAN hotline numbers.
